Containerfile

```diff
@@ -9,6 +9,8 @@ RUN curl -sL https://dl.yarnpkg.com/rpm/yarn.repo -o /etc/yum.repos.d/yarn.repo
                    && dnf -y clean all \
                    && rm -rf /var/cache/yum
 
+ARG APP_PROTOCOL
```

Comment:
Thank you very much for this PR @lucasqueiroz . This fix definitely has the right idea but instead of adding a new env var in the Containerfile, we should localize this fix to where the error happens. 

Can you please give my branch (https://github.com/maestromac/dev.to/tree/mac/container/resolve-imgproxy-startup-issue)  a try and see if it fixed your issue?